ignite/cmd/ignite/main.go

```diff
@@ -36,6 +31,15 @@ func run() int {
 	}
 	defer cleanUp()
 
+	// find command and send to analytics
+	subCmd, _, err := cmd.Find(os.Args[1:])
```

Comment:
Does this remove args to the command, or will it be a separate PR?

Reply:
yes!! remove all args

Reply:
This method will find the command and print the command path instead of the user args